app/src/main/java/dk/scuffed/whiteboardapp/pipeline/Pipeline.kt

```diff
@@ -58,8 +57,15 @@ internal class Pipeline(context: Context, private val initialResolution: Size) :
         GLES20.GL_TEXTURE28,
         GLES20.GL_TEXTURE29,
         GLES20.GL_TEXTURE30,
-        GLES20.GL_TEXTURE31
-        )
+        GLES20.GL_TEXTURE31,
+        GLES20.GL_TEXTURE31 + 1,
+        GLES20.GL_TEXTURE31 + 2
+,
+        GLES20.GL_TEXTURE31 + 3
+,
+        GLES20.GL_TEXTURE31 + 4
+
```

Comment:
Skal dette være med, og det er ikke ens med kommaer.

Reply:
Now it works for any index